x-pack/plugins/maps/public/classes/sources/es_search_source/scaling_form.tsx

```diff
@@ -177,9 +187,49 @@ export class ScalingForm extends Component<Props, State> {
     );
   }
 
+  _renderMVTRadio() {
+    const labelText = i18n.translate('xpack.maps.source.esSearch.useMVTVectorTiles', {
+      defaultMessage: 'Use vector tiles',
+    });
+    const label = <>{labelText}</>;
+    const mvtRadio = (
+      <EuiRadio
+        id={SCALING_TYPES.MVT}
+        label={label}
+        checked={this.props.scalingType === SCALING_TYPES.MVT}
+        onChange={() => this._onScalingTypeChange(SCALING_TYPES.MVT)}
+        disabled={!this.props.supportsMvt}
+      />
+    );
+
+    const info = (
+      <>
+        <EuiBetaBadge label={'beta'} />
+        <EuiHorizontalRule margin="xs" />
+        {i18n.translate('xpack.maps.source.esSearch.mvtDescription', {
+          defaultMessage:
+            'Using .mvt tiles allows for faster display of large datasets. Not all layer-functionality is supported.',
```

Comment:
Change to text to be more assertive. Something like "Use vector tiles for faster display of large datasets. Vector tiles does not support all layer functionality." What layer functionality is not supported with vector tiles? Maybe list them. Is the lack of functionality more then just joins?

Reply:
no joins and degraded legends. 

I'm going to remove this from here. It has a beta-badge. Can add details about trade-offs to the docs.